solutionbox/inception/datalab_solutions/inception/_package.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Provides interface for Datalab. It provides:
```

Comment:
I would expect users to always run it through Datalab, since Datalab provides a more friendly environment for input and output. the package itself is Datalab aware (i.e., in prediction(), if in IPython, display images).

Datalab includes a simple "package runner", which is capable of dumping function args (for help purpose) and passing inputs from input cell to the functions. It is not hard to add the runner to the package so it is callable in the form of "python -m [module]". Or users can run python -c "import datalab_solutions; datalab.solutions.inception.preprocess()" without Datalab.